apps/frontend-manage/.env.qa

```
@@ -10,3 +10,4 @@ NEXT_PUBLIC_MANAGE_URL="https://manage.klicker-qa.bf-app.ch"
 NEXT_PUBLIC_CONTROL_URL="https://control.klicker-qa.bf-app.ch"
 NEXT_PUBLIC_AUTH_URL="https://auth.klicker-qa.bf-app.ch"
 NEXT_PUBLIC_LTI_URL="https://lti.klicker-qa.bf-app.ch"
+NEXT_PUBLIC_CHAT_URL="https://chat.klicker-qa.bf-app.ch"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)